api/namex/services/lookup/name_request_filing_actions.py

```diff
@@ -187,11 +191,15 @@ def get_actions(self, nr_type: str, entity_type_cd: str) -> Optional[dict]:
             URL: the location of the service
             learTemplate: the templat that can be used to create a draft filing.
         """
-        if nr_type in ['FR', 'CFR'] and entity_type_cd in ['FR', 'DBA', 'GP']:
-            nr_type = nr_type + '_' + entity_type_cd
+        if request_type_cd in ['FR', 'CFR'] and entity_type_cd in ['FR', 'DBA', 'GP']:
+            request_type_cd = request_type_cd + '_' + entity_type_cd
 
-        # special case for continued in types that otherwise have the same NR type as regular companies
+        # special case for continued in entity types that otherwise have the same NR type as regular companies
         if entity_type_cd in ['C', 'CBEN', 'CCC', 'CUL']:
-            nr_type = nr_type + '_' + entity_type_cd
+            request_type_cd = request_type_cd + '_' + entity_type_cd
 
-        return self.get_dict.get(nr_type)
+        # special case for continued in request action since entity_type_cd is CR/BC/CC/ULC instead of C/CBEN/CCC/CUL
+        if request_action_cd == 'MVE':
+            request_type_cd = request_type_cd + '_' + entity_type_cd
```

Comment:
I decided to move this special case after the other one, in case Namerequest UI is changed to provide the expected C/CBEN/CCC/CUL entity type.

If Namerequest UI is fixed then the original entries in the table above get used.

While Namerequest UI is not fixed, the extra entries in the table (and this special case) get used.